packages/gatsby-source-contentful/src/download-with-retry.js

```diff
@@ -0,0 +1,40 @@
+const axios = require(`axios`)
+const rax = require(`retry-axios`)
+const { default: PQueue } = require(`p-queue`)
+
+const queue = new PQueue({
+  concurrency: 100,
```

Comment:
Why 100? should we make this configurable through config?

Reply:
We have a limit of 79 requests per second on the Image API. This ensures we do not send to many requests at once and run into rate limiting.

The number was a suggestion from Contentful backend.

Reply:
Additionally: Only a very little number of customers have different rate limits, plus they usually don't know about it. So making this configurable might just cause more confusion

Reply:
could you add a comment that explains what you said :) This would help us not refactor that number in the future